content/applications/productivity/whatsapp.rst

```diff
@@ -0,0 +1,22 @@
+========
+WhatsApp
+========
+
+**WhatsApp** is a popular messaging app that enables users to send texts, make calls, and share
```

Comment:
```suggestion
**WhatsApp** is a popular messaging app that enables users to send texts, and share
```
No calls allow for a business account

Reply:
Ah indeed, thanks, but I thought it'd be interesting to briefly describe the "normal" whatsapp in the first sentence. In the second paragraph, Idon't list calls as features we can do. What do you think?

Reply:
No, it gives a wrong expectation to the customer.
because both are totally different.